system-tests/projects/runner-e2e-specs/cypress/e2e/runner/can-retry-from-afterEach.retries.mochaEvents.cy.js

```diff
@@ -0,0 +1,37 @@
+let i = 0
```

Comment:
I could not get these to work with the `gen-cypress-test` functions we have, so I just wrote these by hand. @tbiethman might have some ideas what I'm missing. I do find these a bit more readable though.

Reply:
I ported the `generateMochaTestsForWin` helper and its usage as-is to minimize potential changes to the existing tests. It's flexible, but to the detriment of its readability, I agree. 

If it doesn't suit your needs, I think writing these out is preferable to further extending the existing APIs.